samples/msmqtosqlrelay/Core_6/MsmqPublisher/MsmqPublisher.csproj

```diff
@@ -41,6 +41,12 @@
     </Reference>
     <Reference Include="NServiceBus.NHibernate, Version=7.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
       <HintPath>..\..\..\..\packages\NServiceBus.NHibernate.7.0.0-beta0002\lib\net452\NServiceBus.NHibernate.dll</HintPath>
+    <Reference Include="NServiceBus.Core, Version=6.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\packages\NServiceBus.6.0.0-beta0003\lib\net452\NServiceBus.Core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="NServiceBus.NHibernate, Version=7.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\packages\NServiceBus.NHibernate.7.0.0-beta0003\lib\net452\NServiceBus.NHibernate.dll</HintPath>
```

Comment:
NServiceBus.NHibernate is referenced twice for some reason (also on line 42 above)